internal/pkg/config/v1/config.go

```diff
@@ -177,14 +181,37 @@ func (c *Config) Load() error {
 		if context.PlatformName == "" {
 			return errors.NewCorruptedConfigError(errors.UnspecifiedPlatformErrorMsg, context.Name, c.Filename)
 		}
-		context.State = c.ContextStates[context.Name]
 		context.Credential = c.Credentials[context.CredentialName]
 		context.Platform = c.Platforms[context.PlatformName]
 		context.Config = c
 		if context.KafkaClusterContext == nil {
 			return errors.NewCorruptedConfigError(errors.MissingKafkaClusterContextErrorMsg, context.Name, c.Filename)
 		}
 		context.KafkaClusterContext.Context = context
+
+		state := c.ContextStates[context.Name]
+		if state != nil {
+			reg, err := regexp.Compile(tokenRegex)
+			if err != nil {
+				return err
+			}
+			if match := reg.MatchString(state.AuthToken); !match && state.AuthToken != mockAuthToken && state.AuthToken != "" { // it's encrypted and not empty
```

Comment:
What's the `mockAuthToken`? Usually we see mocks in tests, not actual code.

Reply:
yes it's used for our integration tests and can be written into the config. during tests i saw sometimes it's actually using the mock context from the config to execute a function.

Reply:
users shouldn't have this problem though

Reply:
any way we can keep this variable out of the code? the testing code should be completely separate from the regular code

Reply:
yeah.. and you might need to delete your config.json first before you can run cli, if you had run make test before